packages/picasso/src/InputAdornment/InputAdornment.tsx

```diff
@@ -21,6 +21,7 @@ export interface Props extends BaseProps, HTMLAttributes<HTMLDivElement> {
   disabled?: boolean
   disablePointerEvents?: boolean
   stopPropagation?: boolean
+  testId?: string
```

Comment:
I'm not sure about this, do we have some rule somewhere to name this `testId`? I would expect
```suggestion
  'data-testid'?: string
```
but I might be wrong.